python/tskit/tables.py

```diff
@@ -1884,6 +1884,30 @@ def sequence_length(self, sequence_length):
     def file_uuid(self):
         return self.ll_tables.file_uuid
 
+    @property
+    def metadata_schema(self) -> metadata.MetadataSchema:
+        """
```

Comment:
Docstring comments aren't indented like this elsewhere.

Reply:
Ditto other new methods here.

Reply:
Dammit, asleep at the wheel today! Fixed.